data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -297,11 +322,14 @@ public void testEqualityDeletesSpanningMultipleDataFiles() throws IOException {
             table, Files.localOutput(temp.newFile()), Row.of(0), dataDeletes, deleteRowSchema);
 
     table.newRowDelta().addDeletes(eqDeletes).commit();
+    // At ths point, the table has (7 - 3) + (8 - 4) = 8 rows. 7 rows in all are deleted.
 
     StructLikeSet expected = rowSetWithoutIds(table, records, 29, 89, 122, 144);
     StructLikeSet actual = rowSet(tableName, table, "*");
 
     Assert.assertEquals("Table should contain expected rows", expected, actual);
+    // 3 deletes in the first data file and 4 deletes in the second data file
```

Comment:
Nit / Non-blocking: this comment seems redundant given we’ve mentioned the 7 deletes previously. But it’s not incorrect so I’m not opposed to keeping it.

Reply:
Removed this comment as above comment has made it clear how many deletes there are.